src/main/java/com/scriptbasic/interfaces/LexicalElement.java

```diff
@@ -79,5 +79,12 @@
      */
     Boolean isSymbol(String lexeme);
 
+    /**
+     * Return true if the lexical element is colon 
```

Comment:
this line is just repetition of the next documentation line and thus useless. It is absolutely okay to have a single `@return` line in a JavaDoc in case of a simple method.